chapter3.adoc

```diff
@@ -32,11 +35,11 @@ The hart's response to abstract commands is detailed in <<sdsecextdbg>>. The fol
 
 ==== Relaxed Permission Check `relaxedpriv`
 
-`relaxedpriv` is hardwired to 0.
+The `relaxedpriv` field is hardwired to 0.
 
 ==== Address Translation `aamvirtual`  
 
-The field `aamvirtual` in command (at 0x17 in Debug Module) determines whether physical or virtual address translation is employed. The extension mandates that `aamvirtual` is hardwire to 1 and memory access addresses are processed as if initiated by the hart i in <<dbgaccpriv, debug access privilege>>.
+The field `aamvirtual` in the command (at 0x17 in the Debug Module) determines whether the Access Memory uses a physical or virtual address. When a physical memory access is issued (`aamvirtual`=0), the hart must check whether it is permitted to conduct such access as described in <<phyaddr>>. If it is not allowed, the hart responds with an exception, and the Debug Module will set `cmderr` to 3 and clear the data registers to 0.
```

Comment:
(Suggsetion) Access Memory => Access Memory Command ; When a Memory Access Command is issued with `aamvirtual` = 0, the hart must check whether the access is permitted as described in <<phyaddr>> and respond an exception to DM if access is not permitted, DM to set `cmderr` to 3 and clear data registers to 0